dune/harmonizer/custom_transforms.py

```diff
@@ -309,7 +309,7 @@ def fix_bytearray_lower(query):
     """Remove lower function call around '0x...' string literals, and remove the string since we have native hex types.
 
     This has to happen after SQLGlot, since it will parse a bare 0x as a string literal"""
-    pattern = r"lower\(\s*['\"]0x(.*?)['\"]\s*\)"
+    pattern = r"lower\(\s*['\"]?0x(.*?)['\"]?\s*\)"
```

Comment:
This will remove `lower()` function calls around 0x strings and 0x literals (i.e. hexstrings). In the future, we should see if we can optimize out the lower() function call using the SQLGlot machinery.

TODO: Update the comment here.

Reply:
https://github.com/duneanalytics/harmonizer/pull/27